codecov.yml

```diff
@@ -7,6 +7,7 @@ ignore:
   - "persist-cli/src/main/java/io/ballerina/persist/components/ClientResource.java"
   - "persist-cli/src/main/java/io/ballerina/persist/components/Client.java"
   - "persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxGenerator.java"
+  - "persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java"
```

Comment:
Do we have to skip this class? Cant we use private constructors?

Reply:
JaCoCo measures test coverage based on the percentage of bytecode. But declaring a static final primitive or string constant does not generate any bytecode to execute. So, we get 0% coverage.

Reference: https://stackoverflow.com/questions/30121265/how-to-add-static-member-variables-jacoco-test-coverage

Reply:
You are still confused on how the code coverage are calculated for constants class. Better to take a look again rather than just fixing the comment w/o understanding.

The number of lines taken into consideration are
1. Inner class 
2. Any static memer other than type of string

Inner classes can never be initiated as this is a constant class. We indicate that by adding private constructors

Reply:
Fixed in [77005fc](https://github.com/ballerina-platform/persist-tools/pull/139/commits/77005fccde26f04e194ee482911af22234cb87d3)